tests/ui/perfherder/alerts_test.jsx

```diff
@@ -252,310 +295,386 @@ const alertsViewControls = () =>
         pathname: '/alerts',
         search: '',
       }}
+      history={createMemoryHistory('/alerts')}
     />,
   );
 
 const modifyAlertSpy = jest.spyOn(mockModifyAlert, 'update');
 
-test('toggle buttons should filter alert summary and alerts by selected filter', async () => {
-  const { getByText, getByTestId } = alertsViewControls();
-  const hideImprovements = getByText('Hide improvements');
-  const hideDownstream = getByText('Hide downstream / reassigned to / invalid');
+describe('Alerts tests', () => {
```

Comment:
We've already grouped the preexisting tests thanks to the `alerts_tests.jsx` filename.
Thus, this `describe` statement isn't DRY. Could we cancel it & leave the tests as they were?

The extra tabs in front of the tests' definition is overwriting a lot of git history & I don't really see a real gain for this.

Reply:
OK, changed.